src/scanner/ethstore.go

```diff
@@ -0,0 +1,239 @@
+package scanner
```

Comment:
This code is also very similar to the btc code, need to make this share the code